src/dispatch/case/views.py

```diff
@@ -161,6 +161,22 @@ def update_case(
     background_tasks: BackgroundTasks,
 ):
     """Updates an existing case."""
+    reporter_email = None
+    if case_in.reporter:
+        # we assign the case to the reporter provided
+        reporter_email = case_in.reporter.individual.email
+    elif current_user:
+        # we fall back to assign the case to the current user
+        reporter_email = current_user.email
+
+    assignee_email = None
+    if case_in.assignee:
+        # we assign the case to the assignee provided
+        assignee_email = case_in.assignee.individual.email
+    elif current_user:
+        # we fall back to assign the case to the current user
+        assignee_email = current_user.email
+
```

Comment:
I think it'd be a better UX if something can be assigned to no one, instead of defaulting to current user, as it could be confusing or not what someone is intending to do (un-assign something completely). 

Should we make reporter/assignee nullable?

Reply:
My thinking was that there shouldn't be any cases w/o a reporter or assignee, but in the event that we had any, then next time someone updated them, they would become the reporter and/ or assignee, so moving forward we would not have those fields empty. I can't think of any cases where we would like cases w/o a reporter and/ or assignee, as we loose context and accountability?